frameworks/PHP/laravel/deploy/roadrunner/.rr.yaml

```diff
@@ -1,23 +1,35 @@
-# see https://roadrunner.dev/docs/intro-config
+# see https://docs.roadrunner.dev/docs/general/config
+# https://github.com/roadrunner-server/roadrunner/blob/master/.rr.yaml
+
 version: "3"
-#https://github.com/roadrunner-server/roadrunner/blob/master/.rr.yaml
 
-server:
-  command: "php ./vendor/bin/rr-worker start --relay-dsn unix:///usr/local/var/run/rr-rpc.sock"
-  relay: "unix:///usr/local/var/run/rr-rpc.sock"
-logs:
-  mode: production
-  level: error
 http:
   address: 0.0.0.0:8080
-  middleware: ["headers"]
   pool:
-    #max_jobs: 64 # feel free to change this
+    num_workers: 0
+    max_jobs: 500 # feel free to change this
```

Comment:
`max-requests` is here

Reply:
Now I'll change to `max_jobs: 0` if later fail in the runs we'll change it again.